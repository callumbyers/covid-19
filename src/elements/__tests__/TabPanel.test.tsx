import * as React from 'react';
import TabPanel from '../TabPanel/TabPanel';
import renderer from 'react-test-renderer';

it('renders correctly', () => {
    const tree = renderer.create(<TabPanel index={0} value={0} />).toJSON();
    expect(tree).toMatchSnapshot();
});