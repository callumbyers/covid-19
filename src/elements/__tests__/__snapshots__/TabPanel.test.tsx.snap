// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<div
  aria-labelledby="simple-tab-0"
  hidden={false}
  id="simple-tabpanel-0"
  role="tabpanel"
>
  <div />
</div>
`;
